draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -496,7 +508,7 @@ struct {
 ```
 
 The fields are described as follows:
-* association_id: The value of the association identifier used to uniquely identify each endpoint in a conference
+* association_id: The value of the association identifier used to uniquely identify each endpoint connected to the media distributor
```

Comment:
Glad you caught this, but technically it identifies the "distinct DTLS association between an endpoint and the key distributor".  If the same endpoint calls into two conferences, each would have a unique association identifier.